### ISSUE TITLE
How to deal with FAMIS data?

### ISSUE THREAD

**Issue by vegetablebird123** [nf-core/proteomicslfq#156]
Hello everyone ，

I am confused that how to edit the experimental design for FAMIS data?
Now  I have raw data with 2 compensation voltage(-45 and -70 v). My experimental design shows below
<img width="434" alt="C2D932E9-68A0-43D2-87E3-648941F70CA7" src="https://user-images.githubusercontent.com/89188780/130042480-538fb43b-82b3-4738-b0f2-db8b90976320.png">
 But the output.csv is not what I expect. For example，It shows the intensity of sample001, -45 and -70v ，respectively. What I expected is every sample gives me 1 intensity, not distinguished by voltage. 
 
So is there anything wrong with my experimental design and how can I fix it? Thanks.

**Comment by jpfeuffer** [comment]
Hi!

That is a good question. Your approach actually could work. But you should probably declare the different voltages as different fractions.

1 1 s1v1.mzML 1
1 2 s1v2.mzML 1
2 1 s2v1.mzML 1
2 2 s2v2.mzML 1
...

**Comment by vegetablebird123** [comment]
It works finally. Thanks！ But you misunderstood me. I do not want to compare intensity between voltages. Actually，I want it gives me one intensity per sample. Can you help me？

**Comment by jpfeuffer** [comment]
Great! Actually, I assumed that already.

> If **not**, try to ..

So, have a look. You should have already intensities per sample.

**Comment by vegetablebird123** [comment]
Here is my out.csv. It also gives me the intensities separated by different voltage per sample. And I find that there are duplicated protein names between one sample with different voltages, for example, 4000 proteins are duplicated in sample1-45v and sample 1-70v. And their intensities are different in 2 references. So which reference should I accept for the  intensity of duplicated protein？thank s. 
![Uploading 9F3B7EC4-A023-4A2B-986A-D30BFF53771F.png…]()

**Comment by vegetablebird123** [comment]
![5EA0D816-8EB5-4160-A8A0-55DCE3AF08C4](https://user-images.githubusercontent.com/89188780/132276489-c54558e1-c5dc-44d0-bb8e-84b0f52f1dd5.png)

**Comment by jpfeuffer** [comment]
Hmm yes, this does not look correct. Can you show your current experimental design?

**Comment by vegetablebird123** [comment]
![6AEA6841-5625-46DE-97D7-F96BF16DBC81](https://user-images.githubusercontent.com/89188780/132298793-e95b7e08-bdb6-4833-8776-2e512b6a70de.png)

Here！

**Comment by vegetablebird123** [comment]
<img width="741" alt="D170392F-1EB5-4FD5-B1FD-685EC4BF41BE" src="https://user-images.githubusercontent.com/89188780/132299058-9cb8b8aa-6ced-4a62-8e58-7ae0c4c8fb71.png">


Here！

**Comment by jpfeuffer** [comment]
Can you try with consecutive numbers starting from 1 in each column?
That means fractions 1,2,1,2,..
And Conditions 1,1,2,2,..

**Comment by vegetablebird123** [comment]
Still the same results.
 
![AC081726-F80D-4129-BB0C-5BE26ABBD102](https://user-images.githubusercontent.com/89188780/132308467-2f90b648-a749-42f8-8b57-14cc3aa1b4fa.png)
![9E34FCAB-33E1-4463-B7DF-A714F41871EE](https://user-images.githubusercontent.com/89188780/132308477-bfc75656-6d1c-4657-8fc2-51ec5ef41123.png)

**Comment by jpfeuffer** [comment]
I disagree. Now it seems like you have the correct annotation of conditions.
What were you expecting?

Do you run the MSstats step? If so, have a look in the msstats/msstats_output.csv. There the intensities should be aggregated further already.

**Comment by jpfeuffer** [comment]
Regarding your "duplication problem": Duplicated proteins are not the problem. This will be common. The only thing that needs to be resolved are peptides with the same sequence, modification and same charge state that might appear at different voltages/fractions.
A common thing to do for fractions is to take the highest intensity feature.

But note that I would let MSstats do the work and just look at the MSstats results as mentioned in my previous comment.
